scripts/bash/validate.sh

```diff
@@ -1,12 +1,7 @@
-#!/bin/bash
-
-set -e
+#!/bin/bash -e
 
 DIR=$(dirname "$0")
 . ${DIR}/../../packages/devops/scripts/ci/utils.sh
 
 yarn -s workspace @tupaia/devops -s validate-branch-name
 yarn -s workspace @tupaia/devops -s validate-tests
-if [[ $? > 0 ]]; then
-    log_warn "Exclusive (.only) tests found in your code. The CI/CD build will fail if you push those"
-fi
```

Comment:
Both previously (with the `set -e`) and now (with the `-e` in the shebang) this code is unreachable

Reply:
Good catch - I guess we shouldn't need this anyway, since `validateTests` logs more specific errors